src/app/components/wallet-deeplink/wallet-deeplink.component.ts

```diff
@@ -83,10 +84,38 @@ export class WalletDeeplinkComponent {
   constructor(private sanitizer: DomSanitizer, private cd: ChangeDetectorRef) {}
 
   private setLinks(): void {
+    this.bloomDeepLink = this.getBloomDeepLink();
     this.fireflyDeepLink = this.getFireflyDeepLink();
     this.tanglePayDeepLink = this.getTanglePayDeepLink();
   }
 
+  private getBloomDeepLink(): SafeUrl {
+    if (!this._network || this._network === Network.IOTA || this._network === Network.ATOI) {
+      // TODO: IOTA network support in Bloom
+      return '';
+    } else {
+      // Shimmer Network
+      const parameters = {
+        address: this.targetAddress,
+        baseCoinAmount:  this.tokenId && this.surplus ? Number(this.targetAmount).toFixed(0): undefined,
+        tokenId: this.tokenId,
+        tokenAmount: this.tokenId ? this.tokenAmount : undefined,
+        tag: WEN_NAME.toLowerCase(),
+        giftStorageDeposit: true,
+        disableToggleGift: true,
+        disableChangeExpiration: true,
+        disableChangeTimelock: true
+      };
+      const searchParametersArray: (string | undefined)[] = Object.entries(parameters).map(([key, value]) => { return value ? `${key}=${value}` : undefined});
+      const searchParametersString = searchParametersArray.filter(x => x !== undefined).flat().join('&');
+
+      return this.sanitizer.bypassSecurityTrustUrl(
+        `bloom://wallet/sendTransaction?${searchParametersString}`
```

Comment:
@Tuditi did we not want to provide a bloom-beta link for rms?

Reply:
We won't have a beta release anytime in the near future, so there is no point to add `bloom-beta`. If we want deep links enabled for alpha releases, I can add `bloom-alpha`. However, we can also use `bloom` deep links for beta releases.

Reply:
Fair enough, if a developer needs to work with alpha version they can easily build both locally